src/components/Input/input.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import "./input.css";
+
+const Input = ({ updateTask, printInput }) => {
+  const [input, setInput] = useState("");
+  const [colorInput, setColorInput] = useState(false);
+
+  const update = (event) => {
+    setInput(event.target.value);
```

Comment:
@PuniaMohit  can this setInput(event.target.value); be done directly? or do you really need a function to setState?